wrappers/services.go

```diff
@@ -131,6 +145,18 @@ func StartServices(apps []*snap.AppInfo, inter interacter) (err error) {
 			}
 		}
 
+		if app.Timer != nil {
+			timerService := filepath.Base(app.Timer.File())
+			// enable the timer
+			if err := sysd.Enable(timerService); err != nil {
+				return err
```

Comment:
I wonder what happens when we have multiple services/timers and fail here (or above on `sysd.Enable(...)`) after already enabling some of them? I guess we've leftovers?

Reply:
The cleanup is done in a defer right here: https://github.com/snapcore/snapd/pull/4696/files#diff-08088787360fb3ca74a0aed4c6103b22R126

Reply:
Right, thanks for reminding me about that (I think the link you gave is not correct, but I see what you mean in the code), thanks.